gallery/src/filterUtil.tsx

```diff
@@ -67,14 +67,22 @@ export function markByFilter(filter: string | undefined, text: string): ReactNod
 /**
  * @return true if the pageName contains every character present in the filter in the same order, case insensitive
  */
-export function matchesFilter(filter: string, pageName: string) {
+export function matchesFilter(filter: string, pageName: string, tags: string[] = []): boolean {
+  const matchesPageName = isStringIncludedInSecondStringCaseInsensitive(filter, pageName);
+  const matchesAnyTags: boolean = map((tag: string) => {
```

Comment:
Ramda has an `any` function that would be simpler than the map/reduce here

Reply:
I figured it had something but struggled to figure it out, I'll poke at that